app/views/issues/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<% content_for :heading do %>
+  <h1><%= t ".title" %></h1>
+<% end %>
+
+<%= form_tag(issues_path, :method => :get) do %>
+<p><%= t ".search_guidance" %></p>
+<%= select :status, nil, Issue.aasm.states.map(&:name).map{|state| [t("issues.states.#{state}"), state]}, { :include_blank => t(".select_status")}, data: { behavior: 'category_dropdown' } %>
+<%= select :issue_type, nil, @issue_types, { :include_blank => t(".select_type")}, data: { behavior: 'category_dropdown' } %>
+<%= text_field_tag :search_by_user, params[:search_by_user], placeholder: t(".reported_user") %>
+<%= select :last_updated_by, nil, @users.all.collect  {|f| [f.display_name, f.id]} << [ t(".not_updated"), "nil"], { :include_blank => t(".select_last_updated_by")}, data: { behavior: 'category_dropdown' } %>
+<%= submit_tag t(".search") %>
+<% end %>
+<br/>
+
+<% if @issues.length == 0 %>
+  <p><%= t ".issues_not_found" %></p>
+<% end %>
+
+<br/>
+
+<table>
+  <thead>
+    <tr>
+      <td><b><%= t ".status" %></b></td>
+      <td><b><%= t ".reports" %></b></td>
+      <td><b><%= t ".reported_item" %></b></td>
+      <td><b><%= t ".reported_user" %></b></td>
+      <td><b><%= t ".last_updated_by" %></b></td>
+      <td><b><%= t ".last_updated_at" %></b></td>
+    </tr>
+  </thead>
+  <tbody>
+    <% @issues.each do |issue| %>
+      <tr>
+        <td><%= t "issues.states.#{issue.status}" %></td>
+        <td><%= link_to t(".reports_count", :count => issue.reports_count), issue %></td>
```

Comment:
Currently this column can, and does, wrap between the number and the word reports. I think it would be better marked as nowrap to stop that.